### ISSUE TITLE
Cannot install on Ubuntu 15.10

### ISSUE THREAD

**Issue by vincentzhezhang** [nulloy/nulloy#89]
Followed the instruction on http://nulloy.com/download/

However, I got error on the last step, seems the package depends on some obsolete lib?
```
sudo apt-get install nulloy nulloy-gstreamer nulloy-taglib 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies.
 nulloy-taglib : Depends: libtag1c2a but it is not installable
E: Unable to correct problems, you have held broken packages.
```

**Comment by noscript** [comment]
Indeed, no idea how I haven't noticed that myself earlier. I will try to fix that asap.

**Comment by noscript** [comment]
Fixed in 1ba3420. Release will be made later today.
